components/03-grid/_grid.scss

```diff
@@ -134,7 +134,9 @@
     @include mq($breakpoint) {
       .col-#{$column}\@#{$breakpoint} {
         width: $width;
-        width: calc(100% / #{$grid-cols / $column});
+        width: -webkit-calc(100% / #{$grid-cols / $column});
```

Comment:
Not sure about this as these will be deprecated in future and stop working.

Worth looking at the original problem using `calc` fixed to see if it's still an issue? Basically, Safari was rendering columns off-pixel because of rounding issues.

Reply:
Yes Safari fixed the rounding issues in 7.1 so think we can remove calc.